src/pgduckdb_types.cpp

```diff
@@ -13,22 +13,21 @@ extern "C" {
 
 #include "pgduckdb/vendor/pg_numeric_c.hpp"
 
-#include "postgres.h"
 #include "fmgr.h"
+#include "postgres.h"
```

Comment:
`postgres.h` should be at the top rest looks fine.

Reply:
gotcha, updated